plutus-playground-client/test/JsonEncodingTests.purs

```diff
@@ -52,13 +52,14 @@ jsonHandlingTests = do
       assertDecodesTo
         (Proxy :: Proxy KnownCurrency)
         "test/known_currency.json"
-    test "Decode a compilation response." do
-      assertDecodesTo
-        (Proxy :: Proxy (InterpreterResult CompilationResult))
-        "generated/compilation_response.json"
-    test "Decode an EvaluationResult." do
-      assertDecodesTo (Proxy :: Proxy (InterpreterResult EvaluationResult))
-        "generated/evaluation_response0.json"
+    -- Note; temporarily disabled
```

Comment:
FIXME?

Reply:
I removed those - i can resurrect them from history when really needed but for now i want to drop them.